src/hotspot/share/gc/g1/g1BarrierSet.cpp

```diff
@@ -107,6 +107,12 @@ void G1BarrierSet::invalidate(JavaThread* thread, MemRegion mr) {
   }
   volatile CardValue* byte = _card_table->byte_for(mr.start());
   CardValue* last_byte = _card_table->byte_for(mr.last());
+  if (G1BarrierSimple) {
+    for (; byte <= last_byte; byte++) {
+      *byte = G1CardTable::dirty_card_val();
+    }
```

Comment:
Why not use memset(), like what we did in dragonwell11?

Reply:
fixed